scripts/rpc-maas-tool.py

```diff
@@ -671,8 +671,9 @@ def _get_failed_checks(self, checks=None):
                 try:
                     result = self.rpcm.conn.test_existing_check(check)
                 except rackspace.RackspaceMonitoringValidationError as e:
-                    validation_error = (" Validation Error: %(s):"
-                                        % {'e': e.message})
+                    validation_error = " Validation Error: {s}:".format(
+                        s=e.message
+                    )
                     break
```

Comment:
@mancdaz yes, I noticed that, I was leaving that to @hughsaunders given this is his patch. I'm not sure if you are implying this should be merged because it is a backport, assuming I'm reading the change correctly, I'd prefer it was abandoned than merged. This patch causes verify-maas to report success even when it's failed, I don't think the bug should be propagated for the sake of consistency.